packages/cevitxe-example-grid/src/ag-grid/emptyGrid.ts

```diff
@@ -1,15 +1,15 @@
 import { JSONSchema7 } from 'json-schema'
 import { range } from 'ramda'
 
-const rowCollectionKey = '::rows'
+const rowCollectionKey = '__rows'
```

Comment:
Mongo doesn't like colons in database names, and I'm guessing they'll cause trouble in other places; so I've just replaced `::` globally with `__`.